### ISSUE TITLE
Source maps do not resolve in dev

### ISSUE THREAD

**Issue by OliverJAsh** [guardian/frontend#8508]
The source maps for `head.content.css` are not resolving in dev. This is because the paths provided in the `sources` key of the source map are relative to the source SCSS file, but of course we do not serve the source files over the HTTP server.

This may be a problem for other files, but I'm not yet familiar enough with our stack to confirm this.

I've tried to make the following changes:

``` patch
diff --git a/Gemfile b/Gemfile
index 8d9e7ea..80afac1 100644
--- a/Gemfile
+++ b/Gemfile
@@ -1,8 +1,8 @@
 source "https://rubygems.org"
 
-gem "sass", "~> 3.3.14"
+gem "sass", "~> 3.4.13"
 
 group :development do
-  gem "scss-lint", "0.26.2"
+  gem "scss-lint", "0.34.0"
   gem "hologram", "1.2.0"
 end
diff --git a/Gemfile.lock b/Gemfile.lock
index 859463d..8a3e9dd 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -5,17 +5,17 @@ GEM
       redcarpet (>= 2.2, < 4.0)
       rouge (>= 1.5)
     rainbow (2.0.0)
-    redcarpet (3.1.2)
-    rouge (1.6.1)
-    sass (3.3.14)
-    scss-lint (0.26.2)
+    redcarpet (3.2.2)
+    rouge (1.8.0)
+    sass (3.4.13)
+    scss-lint (0.34.0)
       rainbow (~> 2.0)
-      sass (~> 3.3.0, >= 3.3.7)
+      sass (~> 3.4.1)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
   hologram (= 1.2.0)
-  sass (~> 3.3.14)
-  scss-lint (= 0.26.2)
+  sass (~> 3.4.13)
+  scss-lint (= 0.34.0)
diff --git a/grunt-configs/sass.js b/grunt-configs/sass.js
index 67268b7..162d643 100644
--- a/grunt-configs/sass.js
+++ b/grunt-configs/sass.js
@@ -4,7 +4,8 @@ module.exports = function(grunt, options) {
     return {
         options: {
             style: 'compressed',
-            sourcemap: options.isDev,
+            sourcemap: options.isDev && 'file',
+            // sourcemap: options.isDev && 'inline',
             noCache: true,
             quiet: options.isDev ? false : true,
             bundleExec: true
```

Embedding the source contents into the source map doesn't seem to help (with the `inline` option, seen above). Neither does providing an absolute file URI (with the `file` option, seen above).

**Comment by OliverJAsh** [comment]
Re-opening as per discussion in https://github.com/guardian/frontend/pull/8526.

Looks like https://github.com/walbo/grunt-px-to-rem/issues/10 was fixed so I will soon have a go at upgrading the Grunt task (thereby fixing the source map issue), unless someone beats me to it.

**Comment by sndrs** [comment]
is this fixed by #8551?

**Comment by sndrs** [comment]
i havn't checked `head.content.css`, so it may not be
